src/app.js

```diff
@@ -1 +1,40 @@
+/* eslint-disable no-console */
 'use strict';
+
+const { createInterface } = require('readline');
+const { generateNumbers } = require('./generateNumbers');
+const { check } = require('./checkInput');
+
+const game = createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const number = generateNumbers();
+
+const runGame = () => {
```

Comment:
```suggestion
const handleRunGame = () => {
```
would be better to start name from `handle`